### DISCUSSION TITLE
Moving Nodes Over to S2

### DISCUSSION THREAD

**Discussion by ioguy** [zwave-js/zwave-js-ui#1734] [Q&A] [answered]
I have 3 door locks (BE469ZP) currently using S0 security.  Is there a way to update them to S2 security without removing and re-adding them?  I'd like to keep the same node IDs and names so that Home Assistant doesn't need to change.

**Comment by robertsLando** [comment, marked as answer]
Actaully there is not an easy way to do that. The best way is to use the raplace failed node to re-include the node with a different security. @blhoward2 do you remember the correct steps to do it?

**Reply by kpine** [reply]
There's a process described here: https://github.com/zwave-js/node-zwave-js/issues/3227#issue-978506479

But you better also shutdown HA (and keep it off) while you do this process. The current versions of HA will remove the device regardless of whether you are replacing it or just excluding it, if it sees a node removal event (emitted for both exclusions and replacements).

**Reply by ioguy** [reply]
Thanks for this - worked great.

The only thing I did different was use a spare USB controller to exclude the device (instead of factory resetting the device) so that I could keep all my device settings.

**Reply by kpine** [reply]
Indeed, a spare USB controller is a much easier way to do this if you have one!
